pkg/gopass/secrets/akv.go

```diff
@@ -298,7 +305,20 @@ func (a *AKV) Body() string {
 
 // Write appends the buffer to the secret's body.
 func (a *AKV) Write(buf []byte) (int, error) {
-	return a.raw.Write(buf)
+	var w io.Writer
+	w = &a.raw
+
+	// If the body is empty before writing we must threat the first line
+	// of the newly written content as the password or multi-line insert
+	// and similar operations will fail. For regular command line usage
```

Comment:
I don't understand what you mean by "and similar operations will fail"?

Reply:
Like insert from STDIN. That should be affected here as well. Updated the comment.